packages/core/src/components/ChoiceList/ChoiceList.jsx

```diff
@@ -81,6 +81,10 @@ export class ChoiceList extends React.PureComponent {
    * @param {array} options - <option> components
    */
   select(selectProps, options) {
+    const sizeClass = this.props.size
+      ? `ds-c-field--${this.props.size}`
+      : undefined;
+    const classes = classNames(sizeClass);
```

Comment:
Do we need the `classNames` line here? Looks like we might be able to get away with just the conditional above since there's one class name possible.